### PR TITLE
Sort versions in download graph using semverSort

### LINKED ISSUE #3245: Versions are sorted alphabetically instead of numerically
**Describe the bug**

1.10.0 is sorted after 1.9.0

**To Reproduce**
1. Go to https://crates.io/crates/iced-x86
2. Scroll down to the download stats
3. 1.10.0 is sorted after 1.9.0

**Expected behavior**

1.10.0 should be sorted before 1.9.0

### PR DESCRIPTION
**Pull request by plaets**
Fixes #3245

### CODE REVIEW

File: app/components/download-graph.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Component from '@glimmer/component';
 
 import subDays from 'date-fns/subDays';
 import window from 'ember-window-mock';
+import semverSort from 'semver/functions/sort';
```
Comment (Turbo87) [inline app/components/download-graph.js:8]: if we import it here we should explicitly add it as a dependency to the `package.json` file :)

**Review by Turbo87 — CHANGES_REQUESTED** [review]

File: app/components/download-graph.js (hunk 1 above)
Comment (plaets) [reply app/components/download-graph.js:8]: You are right, I didn't notice that this is included only in devDependencies. Can I just copy and paste it to dependencies?

Comment (Turbo87) [reply app/components/download-graph.js:8]: yeah, just move it to the `dependencies` section

**Review by Turbo87 — APPROVED** [review]
LGTM! 👍 

@bors r+

**Pull request merged**